### PR TITLE
ci(CodeBuild): run ubuntu latest in series

### PR DESCRIPTION
**Pull request by texastony**
*Issue #, if available:* 
As seen in aws/aws-encryption-sdk-c/pull/746, 
running all our CodeBuild actions at once 
can cause AWS to throttle us.

*Description of changes:*
Execute the CodeBuild Batch Build for `csdk-ubuntu-latest-x64`
serially as compared to in parallel
by using a `build-graph` instead of a `build-list`.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

# Check any applicable:
- [ ] Were any files moved? Moving files changes their URL, which breaks all hyperlinks to the files.

### CODE REVIEW

**Review by alex-chew — COMMENTED** [review]
The batch buildspec changes LGTM, but I'm going to hold off on approving until 746 is merged.

File: codebuild/trusty-gcc4x-x64.Dockerfile
Comment (texastony) [inline codebuild/trusty-gcc4x-x64.Dockerfile:45]:
```suggestion
ENV OPENSSL_TAG=OpenSSL_1_0_2h
```

File: codebuild/trusty-gcc4x-x86.Dockerfile
Comment (texastony) [inline codebuild/trusty-gcc4x-x86.Dockerfile:45]:
```suggestion
ENV OPENSSL_TAG=OpenSSL_1_0_2h
```

**Review by alex-chew — APPROVED** [review]

**Pull request merged**
